scripts/lis_hyperv_platform/main.ps1

```diff
@@ -13,6 +13,9 @@ param(
     [String] $UbuntuImageURL = "https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img",
     [String] $CentosImageURL = "https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud.qcow2",
     [String] $KernelVersionPath = "scripts\package_building\kernel_versions.ini",
+    [String] $PuttyBinaries = "https://the.earth.li/~sgtatham/putty/0.70/w32/putty.zip",
```

Comment:
I would not put these 3 as params, as the probability for them to change is very low.

Reply:
Moreover, is very dangerous to change the params order, as this method might be called using positional params